floor/floor/driver/base.py

```diff
@@ -15,16 +12,11 @@ def __init__(self, driver_args):
         else:
             self.layout = None
 
-    def get_max_led_value(self):
-        return self.MAX_LED_VALUE
-
-    def get_max_floor_value(self):
-        return self.MAX_FLOOR_VALUE
-
     def get_weights(self):
         """
-        Returns the last retrieved list of weight values
-        :return:
+        Returns the last retrieved list of weight values.
+
+        Weights are a value between 0.0 and 1.0.
```

Comment:
It doesn't have to be in this PR, but after sleeping on it I feel like weight values should really be the integers 0 or 1.  Even if the sensors were consistent enough to return the same values for the same step, I can't think of anything meaningful we could do with that in a pattern with 8 people stopping around at once.

Plus the response is pretty non-linear, so most adults are going to drive the value from 0.0 to 0.9.  Getting rid of a nuanced weight value should hopefully simplify things a bit.